src/qml/QFieldSettings.qml

```diff
@@ -273,6 +275,45 @@ Page {
                 Layout.alignment: Qt.AlignTop
             }
 
+            ColumnLayout {
+
+                Label {
+                    padding: 8 * dp
+                    leftPadding: 22 * dp
+                    text: qsTr( "Fast (auto save) mode" )
+                    font: Theme.defaultFont
+
+                    wrapMode: Text.WordWrap
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    Layout.minimumHeight: contentHeight
+                    MouseArea {
+                        anchors.fill: parent
+                        onClicked: autoSaveCheckBox.toggle()
+                    }
+                }
+
+                Label {
+                    padding: 8 * dp
+                    topPadding: 0
+                    leftPadding: 22 * dp
+                    text: qsTr( "If enabled, the feature is stored after having a valid geometry and the constraints are fulfilled and atributes are commited immediately" )
+                    font: Theme.tipFont
+
+                    wrapMode: Text.WordWrap
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    Layout.minimumHeight: contentHeight
+                }
+
+            }
+
+            QfSwitch {
+                id: autoSaveCheckBox
+                checked: true
```

Comment:
The intent is for this new mode to be on by default?

Reply:
No it should be false.